packages/strapi-design-system/src/Toggle/__tests__/Toggle.test.tsx

```diff
@@ -32,32 +32,6 @@ describe('ToggleInput', () => {
     expect(getByRole('checkbox', { name: 'Label' })).toBeDisabled();
   });
 
-  it('should be able to be required', () => {
-    const { getByRole } = render({
-      required: true,
-    });
-
-    expect(getByRole('checkbox', { name: /Label/ })).toBeRequired();
-  });
```

Comment:
I think this is still a valid test?

Reply:
right, I might have misunderstood with label